core/src/banking_stage/transaction_scheduler/scheduler_controller.rs

```diff
@@ -94,11 +96,29 @@ impl SchedulerController {
                 break;
             }
 
+            // update min/max priotization fees
+            let min_max = self.container.get_min_max_prioritization_fees();
+            match min_max {
+                itertools::MinMaxResult::NoElements => {
+                    // do nothing
+                }
+                itertools::MinMaxResult::OneElement(e) => {
+                    min_prioritization_fees = e;
+                    max_prioritization_fees = e;
+                }
+                itertools::MinMaxResult::MinMax(min, max) => {
+                    min_prioritization_fees = min;
+                    max_prioritization_fees = max;
+                }
+            }
```

Comment:
would be ideal if our main loop stayed as simple as possible, let's just pull this into a function so it becomes just a single line: `self.update_container_metrics()`

Reply:
Done